README.md

```diff
@@ -1,27 +1,25 @@
-# The Rust Programming Language
-
-[![Rust Community](https://img.shields.io/badge/Rust_Community%20-Join_us-brightgreen?style=plastic&logo=rust)](https://www.rust-lang.org/community)
+<p align="center">
```

Comment:
If this all looks the same with either the `header` tag or the `section` tag those would be preferable. Also the `align` attribute is deprecated, so assuming it renders the same please write this as
```suggestion
<header style="text-align:center;">
```

Reply:
This is sadly not possible, as GitHub removes any inline style.

We also can't use `<header>` or `<section>` since neither of them supports the `align="center"` hack.

And we can't continue to use `<p>` since it doesn't work with real Markdown links, so I changed it to `<div align="center">`.

Reply:
I "love" this discovery. Apologies, I tried it out in a CommonMark demo before recommending it and didn't realize the [GFM spec](https://github.github.com/gfm/) was out of date.